cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage/CommandLineArg/ForceIndex.hs

```diff
@@ -20,7 +20,7 @@ checkForceIndexesArg =
     startDBSync dbSyncEnv
     -- assertBlockNoBackoff dbSyncEnv 0
     threadDelay 3_000_000
-    assertEqQuery dbSyncEnv DB.queryPgIndexesCount 143 "there wasn't the correct number of indexes"
+    assertEqQuery dbSyncEnv DB.queryPgIndexesCount 161 "there wasn't the correct number of indexes"
```

Comment:
these were failing on me today too (they were out by 1) but if I turned on consumed + purge to True they would pass 🤷

Reply:
There is an additional index when this flag is used, so this is expected
```
CREATE INDEX IF NOT EXISTS idx_tx_out_consumed_by_tx_in_id ON tx_out (consumed_by_tx_in_id)
```